flutter/ios/Classes/SwiftSentryFlutterPlugin.swift

```diff
@@ -4,247 +4,278 @@ import UIKit
-      options.environment = environment
-    }
+        if let releaseName = arguments["release"] as? String {
+            options.releaseName = releaseName
+        }
 
-    if let releaseName = arguments["release"] as? String {
-      options.releaseName = releaseName
-    }
+        if let enableAutoSessionTracking = arguments["enableAutoSessionTracking"] as? Bool {
+            options.enableAutoSessionTracking = enableAutoSessionTracking
+        }
 
-    if let enableAutoSessionTracking = arguments["enableAutoSessionTracking"] as? Bool {
-      options.enableAutoSessionTracking = enableAutoSessionTracking
-    }
+        if let attachStacktrace = arguments["attachStacktrace"] as? Bool {
+            options.attachStacktrace = attachStacktrace
+        }
 
-    if let attachStacktrace = arguments["attachStacktrace"] as? Bool {
-      options.attachStacktrace = attachStacktrace
-    }
+        if let diagnosticLevel = arguments["diagnosticLevel"] as? String, options.debug == true {
+            options.logLevel = logLevelFrom(diagnosticLevel: diagnosticLevel)
+        }
 
-    if let diagnosticLevel = arguments["diagnosticLevel"] as? String, options.debug == true {
-      options.logLevel = logLevelFrom(diagnosticLevel: diagnosticLevel)
-    }
+        if let sessionTrackingIntervalMillis = arguments["autoSessionTrackingIntervalMillis"] as? UInt {
+            options.sessionTrackingIntervalMillis = sessionTrackingIntervalMillis
+        }
 
-    if let sessionTrackingIntervalMillis = arguments["autoSessionTrackingIntervalMillis"] as? UInt {
-      options.sessionTrackingIntervalMillis = sessionTrackingIntervalMillis
-    }
+        if let dist = arguments["dist"] as? String {
+            options.dist = dist
+        }
 
-    if let dist = arguments["dist"] as? String {
-      options.dist = dist
-    }
+        if let enableAutoNativeBreadcrumbs = arguments["enableAutoNativeBreadcrumbs"] as? Bool,
+           enableAutoNativeBreadcrumbs == false {
+            options.integrations = options.integrations?.filter { (name) -> Bool in
+                name != "SentryAutoBreadcrumbTrackingIntegration"
+            }
+        }
 
-    if let enableAutoNativeBreadcrumbs = arguments["enableAutoNativeBreadcrumbs"] as? Bool,
-       enableAutoNativeBreadcrumbs == false {
-      options.integrations = options.integrations?.filter { (name) -> Bool in
-        name != "SentryAutoBreadcrumbTrackingIntegration"
-      }
+        if let maxBreadcrumbs = arguments["maxBreadcrumbs"] as? UInt {
+            options.maxBreadcrumbs = maxBreadcrumbs
+        }
     }
 
-    if let maxBreadcrumbs = arguments["maxBreadcrumbs"] as? UInt {
-      options.maxBreadcrumbs = maxBreadcrumbs
+    private func logLevelFrom(diagnosticLevel: String) -> SentryLogLevel {
+        switch diagnosticLevel {
+        case "fatal", "error":
+            return .error
+        case "debug":
+            return .debug
+        case "warning", "info":
```

Comment:
verbose is more chatty than debug maybe better to put info and warning on debug too?

Reply:
that actually makes sense, not related to this PR though.
I'd even say we should drop `SentryLogLevel` and use only `SentryLevel` which has all the standard sentry levels, but it'd require a change on Cocoa

Reply:
I raised this issue: https://github.com/getsentry/sentry-cocoa/issues/911

Reply:
and this one too https://github.com/getsentry/sentry-dart/issues/277
cc @philipphofmann 
so we don't get blocked by this comment.